config/routes.rb

```diff
@@ -10,4 +10,8 @@
     )
   end
 
+  resources :errors, only: [:show]
+
```

Comment:
Actually, it adds the `error_path` to routes. However, I can use `as: error` on the get path to provide that functionality